src/NServiceBus.NHibernate.AcceptanceTests-SqlTransportTests/NServiceBus.NHibernate.AcceptanceTests-SqlTransportTests.csproj

```diff
@@ -13,14 +13,15 @@
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.1.0" />
     <PackageReference Include="NHibernate" Version="5.3.11" />
     <PackageReference Include="NServiceBus.SqlServer" Version="7.0.0-beta.1" />
-    <PackageReference Include="NServiceBus.AcceptanceTests.Sources" Version="8.0.0-beta.1" GeneratePathProperty="true" />
+    <PackageReference Include="NServiceBus.AcceptanceTests.Sources" Version="8.0.0-beta.2" GeneratePathProperty="true" />
     <PackageReference Include="NUnit" Version="3.13.3" />
     <PackageReference Include="NUnit3TestAdapter" Version="4.2.1" />
     <PackageReference Include="System.Data.SqlClient" Version="4.8.3" />
     <PackageReference Include="GitHubActionsTestLogger" Version="1.3.0" />
   </ItemGroup>
 
   <ItemGroup>
-    <Compile Remove="\**\When_handling_concurrent_messages.cs" />
+    <Compile Remove="$(PkgNServiceBus_AcceptanceTests_Sources)/**/When_handling_concurrent_messages.cs" />
+    <Compile Remove="$(PkgNServiceBus_AcceptanceTests_Sources)/**/When_nested_send_with_outer_replyTo_routing.cs" />
```

Comment:
A comment would be good as why they are excluded

Reply:
👍